content/ru/docs/intro/install.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "Установка Helm"
+description: "Узнайте, как установить и начать работать с Helm."
+weight: 2
+aliases: ["/docs/install/"]
```

Comment:
You can remove all the aliases lines. Those are only necessary for English docs that got moved to a different location. They don't apply to new translations.